Dev bug - get dependencies - using the argument -o without the argument --all-packs-dependencies did not print a proper warning (#1942)
demisto/demisto-sdk: demisto_sdk/commands/find_dependencies/find_dependencies_command.md

* **-ids, --id_set_path** ID set json full path, mainly for skipping creation of id set.
* **--no-update** Use to find the pack dependencies without updating the pack metadata.
* **-v, --verbose** Whether to print the log to the console.
* **--all-packs-dependencies** Return a json file with ALL content packs dependencies. The json file will be saved under the path given in the '--output-path' argument.

> * **-o, --output-path** The destination path for the packs dependencies json file. This argument is only relevant for when using the '--all-packs-dependecies' flag.

Reviewer: ```suggestion
* **-o, --output-path** The destination path for the packs dependencies json file. This argument only works  when using either the `--all-packs-dependencies` or `--get-dependent-on` flags.
```
